MekHQ/src/mekhq/Utilities.java

```diff
@@ -345,10 +348,10 @@ public static int generateExpLevel(int bonus) {
 
         return switch (roll) {
             case 1 -> SkillType.EXP_ULTRA_GREEN;
```

Comment:
Can we remove `SkillType.` here as well?

Reply:
Yes we certainly can - I don't know why that didn't get nixed automatically